packages/block-library/src/post-time-to-read/index.php

```diff
@@ -20,15 +20,31 @@ function render_block_core_post_time_to_read( $attributes, $content, $block ) {
 
 	$content              = get_the_content();
 	$average_reading_rate = isset( $attributes['averageReadingSpeed'] ) ? $attributes['averageReadingSpeed'] : 189;
-	$show_time_to_read    = isset( $attributes['showTimeToRead'] ) ? $attributes['showTimeToRead'] : true;
-	$show_word_count      = isset( $attributes['showWordCount'] ) ? $attributes['showWordCount'] : false;
+
+	// Handle backward compatibility and new displayMode attribute
+	$display_mode = isset( $attributes['displayMode'] ) ? $attributes['displayMode'] : 'time';
+
+	// Backward compatibility: convert old attributes to new displayMode
```

Comment:
#71841 will be included in Gutenberg 21.8. Gutenberg 21.8 is not yet released, so there is no need to worry about backward compatibility.